src/test/groovy/com/synacy/gradprogram/spock/exercise/OrderingServiceSpec.groovy

```diff
@@ -0,0 +1,171 @@
+package com.synacy.gradprogram.spock.exercise
+
+
+import spock.lang.Specification
+import spock.lang.Subject
+
+class OrderingServiceSpec extends Specification {
+
+
+    OrderingService orderingService
+    Cart cart
+
+    def setup() {
+        orderingService = new OrderingService()
+    }
+
+    def "test if the Items will be stored, return true if all the items were stored"(){
+     given:
+        def items = [
+                new Item("Charger", 100.00, ItemType.GADGET),
+                new Item("Vanz Tshirt", 160.00, ItemType.CLOTHING),
+                new Item("Oven", 800.00, ItemType.APPLIANCE),
+                new Item("555 Tuna", 25.00, ItemType.FOOD),
+                new Item("Pork Chop", 150.00, ItemType.FOOD)
+        ]
+        when:
+        boolean ifcontainsFoodType = orderingService.cartContainsFoodItem(cart = new Cart(UUID.randomUUID(), items))
+
+        then:
+        ifcontainsFoodType
+    }
+
+    def "Should Determine the Total Cost of the Cart is true"(){
+        given:
+        def totalItems = [
+                new Item("555 Tuna", 50.00, ItemType.FOOD),
+                new Item("Pork Chop", 150.00, ItemType.FOOD),
+                new Item("Charger", 100.00, ItemType.GADGET),
+                new Item("Vanz Tshirt", 160.00, ItemType.CLOTHING),
+                new Item("Oven", 800.00, ItemType.APPLIANCE)
+        ]
+        when:
+        double totalPrice = orderingService.calculateTotalCostOfCart(cart = new Cart(UUID.randomUUID(),totalItems))
+
+        then:
+        println(totalPrice)
+        totalPrice == 1260.00
+
+    }
+
+
+    def "Should return the items is Eligible for a Discount"(){
```

Comment:
Maybe something like "isCartEligibleForDiscount should return true if the Cart is eligible for discount" would be better.